R/uvhydrograph-render.R

```diff
@@ -437,6 +436,40 @@ getPrimaryPlotConfig <- function(primaryPlotData, plotStartDate, plotEndDate, pr
   return(plotConfig)
 }
 
+#' Get Corrections Plot Config
+#' @description Given corrections, some information about the plot to build, will return a named list of gsplot elements to call
+#' @param corrections list of data objects relavant to plotting corrections data
+#' @param plotStartDate start date of this plot 
+#' @param plotEndDate end date of this plot
+#' @param label label of that these corrections are associated with
+#' @param limits list of lims for all of the data which will be on here
+#' @importFrom grDevices rgb
+getCorrectionsPlotConfig <- function(corrections, plotStartDate, plotEndDate, label, 
+    limits) {
+  
+  if(nrow(corrections) <= 0) {
```

Comment:
Just a single return statement would be preferred for consistency - this would have `plotConfig <- list()`. I also think you can use `if(isEmptyVar(corrections))`